test/e2e/snaps/test-snap-multi-install.spec.js

```diff
@@ -0,0 +1,111 @@
+const {
+  defaultGanacheOptions,
+  withFixtures,
+  unlockWallet,
+  WINDOW_TITLES,
+} = require('../helpers');
+const FixtureBuilder = require('../fixture-builder');
+const { TEST_SNAPS_WEBSITE_URL } = require('./enums');
+
+describe('Test Snap Multi Install', function () {
+  it('test multi install snaps', async function () {
+    await withFixtures(
+      {
+        fixtures: new FixtureBuilder().build(),
+        ganacheOptions: defaultGanacheOptions,
+        failOnConsoleError: false,
+        title: this.test.fullTitle(),
+      },
+      async ({ driver }) => {
+        await unlockWallet(driver);
+
+        // navigate to test snaps page and multi-install snaps
+        await driver.openNewPage(TEST_SNAPS_WEBSITE_URL);
+        await driver.delay(1000);
+        const dialogButton = await driver.findElement('#multi-install-connect');
+        await driver.scrollToElement(dialogButton);
+        await driver.delay(1000);
```

Comment:
Can this delay be replaced with something using `await driver.waitForSelector`?

Reply:
The current framework scrollToElement dialogs don't work with a waitForSelector. I think it's because of the wait that waitForSelector is constructed as it attempts to find the selector before it's visible in the window.